presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerifierConfig.java

```diff
@@ -314,7 +315,7 @@ public boolean isSetupOnMainClusters()
         return setupOnMainClusters;
     }
 
-    @ConfigDescription("If true, run control/test setup queries on control/test clusters. Otherwise, run setup queries on the help cluster.")
+    @ConfigDescription("If true, run control/test setup queries on control/test clusters. Otherwise, run setup queries on the help cluster.X")
```

Comment:
```suggestion
    @ConfigDescription("If true, run control/test setup queries on control/test clusters. Otherwise, run setup queries on the help cluster.")
```